src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -324,26 +372,27 @@ private org.candlepin.subscriptions.db.model.Subscription convertDto(Subscriptio
         .build();
   }
 
-  protected void updateSubscription(
-      Subscription dto, org.candlepin.subscriptions.db.model.Subscription entity) {
-    if (dto.getEffectiveEndDate() != null) {
-      entity.setEndDate(clock.dateFromMilliseconds(dto.getEffectiveEndDate()));
-    }
+  private static org.candlepin.subscriptions.db.model.Subscription convertDto(
+      org.candlepin.subscriptions.umb.Subscription subscription) {
```

Comment:
Probably a topic for dev sync but I think it might be helpful to name these more uniquely to avoid conflicts. (SubscriptionUmb, SubscriptionDb, etc..)

Reply:
Nah, I like the idea of just doing it now. Will do s/Subscription/UmbSubscription/

Reply:
Awesome sounds good :)